incubator/binding-mqtt.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/schema/mqtt.schema.patch.json

```diff
@@ -32,7 +32,63 @@
                         "enum": [ "server" ]
                     },
                     "vault": false,
-                    "options": false,
+                    "options":
+                    {
+                        "properties":
+                        {
+                            "authorization":
+                            {
+                                "title": "Authorizations",
+                                "type": "object",
+                                "patternProperties":
+                                {
+                                    "^[a-zA-Z]+[a-zA-Z0-9\\._\\-]*$":
+                                    {
+                                        "title": "Authorization",
+                                        "type": "object",
+                                        "properties":
+                                        {
+                                            "credentials":
+                                            {
+                                                "title": "Credentials",
+                                                "type": "object",
+                                                "properties":
+                                                {
+                                                    "connect":
+                                                    {
+                                                        "title": "Connect",
+                                                        "type": "object",
+                                                        "additionalProperties":
+                                                        {
+                                                            "type": "string",
+                                                            "pattern": ".*\\{credentials\\}.*"
+                                                        }
```

Comment:
This is not actually freeform, should be explicitly `username` or `password`, right?